packages/ui/src/components/Button/index.tsx

```diff
@@ -306,9 +317,12 @@ export const Button = forwardRef<Element, FinalProps>(
         {isLoading ? (
           <Loader
             active
-            trailColor="transparent"
-            size="1em"
-            color="currentColor"
+            size="small"
+            sentiment={
+              variant === 'filled' && !['black', 'white'].includes(sentiment)
+                ? 'neutral'
```

Comment:
wouldn't it work better with a sentiment white instead? I feel like the loading is not much visible